### PR TITLE
Add images ingestor to the pipeline

### PR DESCRIPTION
**Pull request by alicefuzier**

### CODE REVIEW

File: pipeline/terraform/main.tf [hunk 1]
```diff
@@ -40,7 +40,72 @@ module "catalogue_pipeline_20200228" {
...
+    local.mets_adapter_topic_arn,
+  ]
+
+  calm_adapter_topic_arns = [
+    local.calm_reindexer_topic_arn,
+    local.calm_adapter_topic_arn,
+  ]
+
+  # Elasticsearch
+  es_works_index  = "works-20200414"
```
Comment (jamesgorrie) [inline pipeline/terraform/main.tf:107]: [We seem to have two of these now](https://github.com/wellcomecollection/catalogue/pull/529/files#diff-abff2a9b44708c7537e4a5e72b530571R43-R44)?

File: pipeline/terraform/stack/release_uris.tf [hunk 2]
```diff
@@ -30,6 +31,7 @@ locals {
   inference_manager_image  = local.image_ids["inference_manager"]
   feature_inferrer_image   = local.image_ids["feature_inferrer"]
   ingestor_works_image     = local.image_ids["ingestor_works"]
+  ingestor_images_image    = local.image_ids["ingestor_images"]
```
Comment (jamesgorrie) [inline pipeline/terraform/stack/release_uris.tf:34]: 🌀

File: pipeline/terraform/main.tf (hunk 1 above)
Comment (alicefuzier) [reply pipeline/terraform/main.tf:107]: Yes, the other one is for the old pipeline. I didn't want to destroy it just yet

Comment (jamesgorrie) [reply pipeline/terraform/main.tf:107]: Misread the code 🔍

**Pull request merged**